libraries/scopes/src/simplify.js

```diff
@@ -0,0 +1,2 @@
+exports.simplifyScopeExpression = (scope) =>
+  scope === 'First test' || 'Second test';
```

Comment:
The first step is to create a function that returns the correct *type* of data but without any logic.  What you've written here returns a boolean or a string, but the function should return a scope expression.